lab7-record.md

```diff
@@ -488,9 +488,9 @@ List<Speaker> allSpeakers = speakers.stream()
 
 ## Wrap-up
 
-In this exercise, you have used **records**, a new feature that enables to easily create immutable data aggregates. The record feature went through 2 preview rounds (Java 14 & Java 15) and are slated to be made final and permanent in Java 16.
+In this exercise, you have used **Records**, a new feature for the easy creation of immutable data aggregates, but also provides many other benefits! The Record feature went through 2 preview rounds (Java 14 & Java 15) and was made final and permanent in Java 16.
```

Comment:
does ", but also provides..." sounds right?

Reply:
Yea you are right, let me re-work that a bit.